internal/proxy/hook/authz/authz_test.go

```diff
@@ -37,18 +37,16 @@ var expectedResources = []resource.Resource{
 		ProjectID:      "ab657ae7-8c9e-45eb-9862-dd9ceb6d5c71",
 		OrganizationID: "org1",
 		Name:           "resc1",
-		NamespaceID:    "ns1_kind",
+		NamespaceID:    "ns1/kind",
 	}, {
 		ProjectID:      "ab657ae7-8c9e-45eb-9862-dd9ceb6d5c71",
 		OrganizationID: "org1",
 		Name:           "resc2",
-		NamespaceID:    "ns1_kind",
+		NamespaceID:    "ns1/kind",
 	},
 }
 
 func TestCreateResources(t *testing.T) {
```

Comment:
Please recheck why these changes are necessary, don't seem relevant for the PR.

Reply:
Yes, these are not for this problem. These tests had this wrong configuration, due to which they are failing. Not sure how they were working earlier in parallel mode.